examples/carousel/carousel-2-tablist.html

```diff
@@ -0,0 +1,779 @@
+    <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+    <link rel="stylesheet" href="../css/core.css">
+    <script src="../js/examples.js"></script>
+    <script src="../js/highlight.pack.js"></script>
+    <script src="../js/app.js"></script>
+
+    <!--  js and css for this example. -->
+    <script type="text/javascript">
+      var stylesheet = 'css/carousel-tablist.css';
+      if (location.href.indexOf('moreaccessible') > 0) {
+        console.log('loaded more accessible');
+        stylesheet = 'css/carousel-tablist-more-accessible.css';
+      }
+
+      document.write('<link rel="stylesheet" type="text/css" href="' + stylesheet + '">')
+
+      window.addEventListener('load', function () {
+        if (location.href.indexOf('moreaccessible') > 0) {
+          document.getElementById('id_link_more_accessible').setAttribute('aria-current', 'page');
+        }
+        else {
+          document.getElementById('id_link_less_accessible').setAttribute('aria-current', 'page');
+        }
+      });
+
+    </script>
+
+    <script src="js/carousel-tablist.js" type="text/javascript"></script>
+
+  </head>
+  <body>
+    <nav aria-label="Related Links" class="feedback">
+      <ul>
+        <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/projects/10">Related Issues</a></li>
+        <li><a href="../../#carousel">Design Pattern</a></li>
+      </ul>
+    </nav>
+    <main>
+      <h1>Example of Image Carousel with Automatically Rotating Images using a Tablist</h1>
+        <p>
+          Comments on this example are tracked in
+          <a href="https://github.com/w3c/aria-practices/pull/1120">Pull request 1120.</a>
+        </p>
+      <p>
+        The following example implementation of the
+        <a href="../../#carousel">carousel design pattern</a>
+        demonstrates features of the pattern that are essential to accessibility for carousels that automatically start rotating when the page loads and allow users to directly select which slide is presented.  Slide selction is typically provided using a series of circular buttons rendered over the rotating slides.
+        Automatic rotation stops when users move focus to any control or link in the carousel or hover the mouse over carousel content.
+        The accessibility features section that follows the example describes these features in detail.
+      </p>
+
+      <section>
+        <h2>Example View Options</h2>
+        <ul>
+          <li><a id="id_link_less_accessible" href="carousel-2-tablist.html">Tablist carousel controls and captions displayed on image</a>: This option is less accessible than the other option because the controls are harder to perceive and captions are more difficult to read with rotating images behind them.</li>
```

Comment:
```suggestion
          <li><a id="id_link_less_accessible" href="carousel-2-tablist.html">Tablist carousel controls and captions displayed on image</a>: This option is less accessible than the one below because controls are harder to perceive and captions are more difficult to read with rotating images behind them.</li>
```

Reply:
@carmacleod fixed